mempool/mempool.go

```diff
@@ -189,6 +189,11 @@ type Policy struct {
 	// admitted and relayed.
 	AllowOldVotes bool
 
+	// MaxVoteAge defines the number of blocks in history from the next block
+	// height of the best chain tip for which votes will be accepted.  This only
+	// applies when the AllowOldVotes option is false.
+	MaxVoteAge func() uint16
```

Comment:
No need to make this a function that has to be invoked on every transaction.  It only needs to be setup once at initialization time.

`MaxVoteAge uint16`